app/views/play/level/modal/CourseVictoryModal.coffee

```diff
@@ -170,9 +170,6 @@ module.exports = class CourseVictoryModal extends ModalView
       session: @session.toJSON(),
       course: @course.toJSON(),
       @courseInstanceID,
-      stats: @classroom?.statsForSessions(@levelSessions, @course.id)
```

Comment:
This is used in the vue component here:
https://github.com/codecombat/codecombat/blob/5a5787a/app/views/play/level/modal/CourseVictoryComponent.vue#L69

Reply:
It seems fine to remove this and avoid calculating it.  Can ask @ryang217 whether to remove for IE11-only or all browsers.